misc/python/materialize/checks/roles.py

```diff
@@ -11,9 +11,13 @@
 
 from materialize.checks.actions import Testdrive
 from materialize.checks.checks import Check
+from materialize.util import MzVersion
 
 
 class CreateRole(Check):
+    def _can_run(self) -> bool:
+        return self.base_version >= MzVersion(0, 44, 999)
```

Comment:
Why 0.44.999 instead of 0.45.0?

Reply:
for some reason the current version from `main`, 0.45-dev is considered to be less than 0.45 so the test will be skipped and we want it to run for the current version, just not for any previous version.

Reply:
Ah right, semver does that, because 0.45.0 is considered to be the released version, while 0.45.0-dev is a beta version before 0.45.0. It makes sense but is a bit unfortunate. `MzVersion(0, 45, 0, 'dev')` or `MzVersion.parse_mz('0.45.0-dev')` should work and might be slightly clearer.

Reply:
Thanks, fixed.